packages/app-mobile/components/ComboBox.tsx

```diff
@@ -63,22 +63,25 @@ interface UseSearchResultsOptions {
 const useSearchResults = ({
 	search, setSearch, options, onAddItem, canAddItem,
 }: UseSearchResultsOptions) => {
+	const collatorLocale = getCollatorLocale();
+	const collator = getCollator(collatorLocale);
```

Comment:
Consider wrapping this in a `useMemo` to prevent unnecessary recomputing of `results`:
- Each call to `getCollator` returns a `new Intl.Collator`. 
- [React uses Object.is](https://react.dev/reference/react/useMemo#skipping-expensive-recalculations) for dependency comparisons.
- ```js
  const makeCollator = () => new Intl.Collator('en-US', { numeric: true, sensitivity: 'accent' });
  console.log(Object.is(makeCollator(), makeCollator()));
  ```
  prints `false`.

This change might look similar to one of the following:
- ```suggestion
  	const collatorLocale = getCollatorLocale();
  	const collator = useMemo(() => {
  		return getCollator(collatorLocale);
  	}, [collatorLocale]);
  ```
- Or, alternatively,
  ```suggestion
  	const collator = useMemo(() => {
  		const collatorLocale = getCollatorLocale();
  		return getCollator(collatorLocale);
  	}, []);
  ```

Reply:
Ah, I did wonder why getCollator was inside useMemo in TagList.tsx and thought it might have been suboptimal. I've now updated this